### ISSUE TITLE
Question: how i login or register in idurar web application

### ISSUE THREAD

**Issue by MugishBeldar** [idurar/idurar-erp-crm#896]
Questions
login:- how l login in idurar web application
Registration:- when i do register on idurar application i face this error (Request error 409 Registration is not allowed , please contact application administrator)

**Comment by Prabhat-6789** [comment]
first you will have to do setup codebase. I think you have not run command "**npm run setup**". Therefore this issue came.

**Comment by parth-chovatiya** [comment]
1. Setup frontend & backend codebase
2. Setup the database key (in .env) 
3. Go to the backend directory, run the below command on terminal
    i.  `npm install`
    ii. `npm run setup`
4. Go to a database in the **settings** collection and find the document having **settingKey: idurar_registration_allowed**. Change the **settingValue from false to true**

**Comment by ThibaltZer** [comment]
I modified settingKey: idurar_registration_allowed

I see my account has been registered in the database but when I log in I get the following error
Request error 500
` Missing API key. Pass it to the constructor new Resend("re_123")` 

I try npm run setup again 
` MongoServerError: E11000 duplicate key error collection: mydatabase.admins index: email_1 dup key: { email: "admin@demo.com" }`

**Comment by Prabhat-6789** [comment]
You will have to first geneate an `API KEY` on **resend** and then set `API KEY` as environment variable in .env file.

**Comment by ThibaltZer** [comment]
Thanks a lot

I have configured the resend api key 
I have validated my domain on resend 

But I see a 403 error with the following message 
"message": "The idurarapp.com domain is not verified.

Where can I change the domain in the configuration files?

**Comment by WarrenMartin** [comment]
yes even i am facing the issue even on the youtube video it isnt told properly please help us

**Comment by ratnabh** [comment]
Hi @ptminh-kmp can you please elaborate what do you mean by **Access to MongoDb, schema "settings" and update the following settingKey:** , I am not aware about mongodb, and how to set it up :(

**Comment by newetard** [comment]
hi, im able to register, but unable to login. can someone please help
